### DISCUSSION TITLE
Missing Region column on Devices object

### DISCUSSION THREAD

**Discussion by Nino-Rey** [netbox-community/netbox#10143] [Q&A]
Hi All,

Greetings!

I noticed that Region column is missing in Table configuration for Devices object. But inside the device the Region is there. Need help on how to populate the Region column.

![image](https://user-images.githubusercontent.com/93611480/186553980-dccf69de-2ee8-42c7-8171-7b97c2d4961a.png)

![image](https://user-images.githubusercontent.com/93611480/186554202-f122f545-b065-4fbf-9108-3cba872e86d8.png)

![image](https://user-images.githubusercontent.com/93611480/186554335-8db78034-2ed7-4688-8d6d-b06f11f297a8.png)

**Comment by abhi1693** [comment]
As per the code, it's a computed field because you have a site associated with the device. Currently, this is not possible but you can raise a FR with your use case to get this added.

**Reply by Nino-Rey** [reply]
Thanks @abhi1693 . I just raise an FR.
